game/environment.go

```diff
@@ -166,15 +186,22 @@ func (g *environment) StartGame(tag, name string, scenarioNo int) error {
 	numNetworks := len(selectedScenario.Networks)
 	log.Info().Msgf("Setting openvswitch bridge %s", bridgeName)
 	if err := g.initializeOVSBridge(bridgeName); err != nil {
-		return err
+		log.Error().Err(err).Msg("Error with initializing OVSBridge")
+		//return err
 	}
 	if err := g.createRandomNetworks(bridgeName, int(numNetworks)); err != nil {
-		return err
+		log.Error().Err(err).Msgf("Error on creating random networks")
+
+	}
+
+	if err := g.configureMonitor(bridgeName, numNetworks); err != nil {
+		log.Error().Err(err).Msgf("Error to configure monitoring")
 	}
 
 	if err := g.initializeScenarios(bridgeName, &g.controller, scenarioNo); err != nil {
-		return err
+		log.Error().Err(err).Msgf("Error on initializing scenarios")
 	}
```

Comment:
No return err ? Is there specific reason for it ?

Reply:
From my best understanding, the error is returned here: 
`log.Error().Err(err).Msgf("Error on initializing scenarios")`

I have seen when running the code that I get the error message. @Lanestolen Can you comment here, it was your advice :)

Reply:
Hey, no the error isn't return, how ever the error will be shown as a sperate field in the output json